### DISCUSSION TITLE
Module Discoverability

### DISCUSSION THREAD

**Discussion by briandowns** [Jason2605/Dictu#722] [Ideas]
Was hoping to get the conversation started around user created/3rd party module discovery. We have a number of options. Here are the ones I can think of.

* Create repos in the dictu-lang org for 'dictu' sanctioned modules.
* Create a service like PyPi
* Use dictum without any structure. ie `dictum install github.com/briandowns/kahless v0.1.1`
* Hybrid of 2 and 3?
* Something else?

**Comment by Jason2605** [comment]
Hey! Hope you have had a good bit of time off so far and look forward to new years tonight!

I think the easiest for adoption will probably be point 3 with point 2 coming probably (a lot) later. Point 1 I also like yeah with the concept of "official" modules that just live in the organisation (feel free to make any repos in there by the way - I *think* you should have access)

**Comment by briandowns** [comment]
Was also thinking we can use the wiki section on here to list out modules similar to how wren does on their wiki.

**Comment by briandowns** [comment]
Not so much in regards to discovery but in case anyone might be interested in a module I've been using lately that I've found super helpful:

https://github.com/briandowns/mapledb

**Comment by briandowns** [comment]
I fleshed out the wiki with a bunch of stuff to at least get a start. Please feel free to edit, delete or whatever with it. https://github.com/dictu-lang/Dictu/wiki

**Reply by Jason2605** [reply]
Nice one, appreciate it!!
